waspc/cli/exe/Main.hs

```diff
@@ -238,21 +240,25 @@ printDbUsage =
               "  wasp db <command> [command-args]",
               "",
         title "COMMANDS",
-        cmd   "  start         Alias for `wasp start db`.",
-        cmd   "  reset         Drops all data and tables from development database and re-applies all migrations.",
-        cmd   "  seed [name]   Executes a db seed function (specified via app.db.seeds).",
-              "                If there are multiple seeds, you can specify a seed to execute by providing its name,",
-              "                or if not then you will be asked to provide the name interactively.",
         cmd $ intercalate "\n" [
-              "  migrate-dev   Ensures dev database corresponds to the current state of schema(entities):",
-              "                  - Generates a new migration if there are changes in the schema.",
-              "                  - Applies any pending migrations to the database either using the",
-              "                    supplied migration name or asking for one.",
+              "  start [--db-image <image>]   Alias for `wasp start db`.",
+              "                               Starts managed development database for you.",
+              "                               Optionally specify a custom Docker image."
```

Comment:
Not the same as for `wasp start db` (missing "with --db-image). I would add it here or remove it up there.